src/shell.js

```diff
@@ -235,7 +235,9 @@ if (ENVIRONMENT_IS_NODE) {
   // EXPORT_ES6 + ENVIRONMENT_IS_NODE always requires use of import.meta.url,
   // since there's no way getting the current absolute path of the module when
   // support for that is not available.
-  scriptDirectory = require('url').fileURLToPath(new URL('./', import.meta.url)); // includes trailing slash
+  // Using Object(import.meta).url here to prevent Webpack from trying to resolve the URL
+  // https://github.com/webpack/webpack/issues/16878
+  scriptDirectory = require('url').fileURLToPath(new URL('./', Object(import.meta).url)); // includes trailing slash
```

Comment:
OMG, this is pretty disgusting.    

It makes me wonder if we need some kind of flag such as `WEBPACK_COMPATIBLE` that that we don't need to subject all users to this. 

However, I suppose that closure compiler can/will just remove this so maybe is not a huge deal.

Reply:
I really don't like bundlers, but a lot of people do, so I'd like to do something.

What do you think is best?

Reply:
I think `path.dirname(fileURLToPath(import.meta.url);` would be great it if works.  Is webpack ok with it?

Reply:
@sbc100 Yea, that notation works.